src/overlays/actors/ovl_Demo_Tre_Lgt/z_demo_tre_lgt.c

```diff
@@ -23,21 +35,136 @@ const ActorInit Demo_Tre_Lgt_InitVars = {
     (ActorFunc)DemoTreLgt_Update,
     (ActorFunc)DemoTreLgt_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Demo_Tre_Lgt/DemoTreLgt_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Demo_Tre_Lgt/DemoTreLgt_Destroy.s")
+static TransformUpdateIndex* sTransformUpdIdx[] = { 0x06004B60, 0x06004F70 };
+
+static DemoTreLgtActionFunc sActionFuncs[] = {
+    func_8099375C,
+    func_80993848,
+};
+
+void DemoTreLgt_Init(Actor* thisx, GlobalContext* globalCtx) {
+    DemoTreLgt* this = THIS;
+
+    if (!SkelCurve_Init(globalCtx, &this->skelCurve, &D_06005EB8, sTransformUpdIdx[0])) {
+        osSyncPrintf("Demo_Tre_Lgt_Actor_ct();コンストラクト失敗\n");
+    }
+
+    // defined in .rodata but not used
+    "1";
+    "../z_demo_tre_lgt.c";
```

Comment:
I suggest putting an assert here since that's very likely what it was.
It would go something like this (UNK_LINE already exists):
```suggestion
    // This assert is optimized out but it exists due to its presence in rodata
    if (0) {
        __assert("1", "../z_demo_tre_lgt.c", UNK_LINE);
    }
```